README.md

```diff
@@ -68,6 +68,8 @@ Setup steps:
 1. Disable *SAML\_ENABLED* auth and enable *BASIC\_ENABLED*
 1. Edit /etc/hosts file __OPTIONAL__
   * Add _web_ and _api_ to the end of the localhost line in /etc/hosts so you don't have to keep changing the values of API\_REDIRECT or API\_BASE\_URL in .env file back to localhost
+1. Run __make run__ to ensure services are restarted
+1. Run the purge directions from docs/test-scenarios.md to clear out old data
```

Comment:
@gidjin Markdown doesn't seem to be numbering the steps as you would expect:

<img width="400" alt="screen shot 2018-12-17 at 1 56 44 pm" src="https://user-images.githubusercontent.com/37100189/50118158-b4069b00-0203-11e9-94af-6ac9ac3ec02e.png">

Reply:
Apparently not. I've updated it, I couldn't get sublists to number properly so I took them out since it always started over.